doc/mgr/smb.rst

```diff
@@ -78,6 +78,9 @@ domain_realm
 domain_join_user_pass
     A string in the form ``<username>%<password>`` that will be used to join
     Samba servers to the AD domain.
+define_user_pass
+    A string in the form ``<username>%<password>`` that will be used for
```

Comment:
Suggest s/in/of/

Reply:
We'll pick this up in a later PR.

Reply:
> Suggest s/in/of/

I've incorporated this in the latest version.